vue.config.js

```diff
@@ -0,0 +1,7 @@
+const config = require('@rancher/shell/vue.config');
+
+module.exports = config(__dirname, {
+  excludes: [],
+  // excludes: ['fleet', 'example']
+  // autoLoad: ['fleet', 'example']
```

Comment:
You can get rid of autoload https://github.com/rancher/dashboard/pull/8700#event-9115687142